sleap/io/cameras.py

```diff
@@ -2331,6 +2381,7 @@ def upsert_points(
                 points=instance_points,
                 cams_to_include=self.cams_to_include,
                 exclude_complete=exclude_complete,
+                bounds=bounds,
```

Comment:
Make sure this input name matches the name you decided on for projection bounds in the definition for `InstanceGroup.update_points`